content/03.installation.md

```diff
@@ -1,6 +1,61 @@
-# Install
+## Installation
 
-There are two ways to install CNApy:
+### CNApy main program
 
-1. Under any operating system, by installing CNApy as a conda package (see [https://github.com/cnapy-org/CNApy#install-cnapy-with-conda](https://github.com/cnapy-org/CNApy#install-cnapy-with-conda))
-2. As an alternative installation way under Windows, by using the .exe installer attached to [CNApy's latest release](https://github.com/cnapy-org/CNApy/releases/latest).
+There are two ways to install CNApy itself:
+
+1. Recommended but more complicated: Under any operating system, by installing CNApy as a conda package. For detailed instructions to do this, see [https://github.com/cnapy-org/CNApy#install-cnapy-with-conda](https://github.com/cnapy-org/CNApy#install-cnapy-with-conda).
+2. Not recommended but simple: Under Windows, you can install CNApy by using the .exe installer attached to [CNApy's latest release](https://github.com/cnapy-org/CNApy/releases/latest). Once you run the installer, it will guide you through the installation process.
+
+
+### CNA legacy bridge requirements (optional)
+
+In order to use "old" function from the original Matlab/Octave *CellNetAnalyzer*, you need one of the following programs installed at your systems (follow the links to the programs for download and installation instructions)...
+1. [MATLAB](https://www.mathworks.de/products/matlab/index.html), **in a version from 2019b up to 2021b (older and newer versions are not supported!)**.
+2. [GNU Octave](https://www.gnu.org/software/octave/index) installed at your system, **in a version >6.4.0 (older versions are not supported!)**.
+
+...as well as the original [*CellNetAnalyzer*](http://www2.mpi-magdeburg.mpg.de/projects/cna/cna.html/).
+
+
+### Additional solvers (optional)
+
+*Note: Additional solvers are only possible if you've installed CNApy according to the first main method, i.e. by using conda*
+
+CNApy itself is already packaged and installed with the open source linear programming solver [GLPK](https://www.gnu.org/software/glpk/) which is fast enough for simple calculations.
+However, for large models and more complications calculations (such as Minimal Cut Set analyses), it is recommended to install a more sophisticated and faster solver such as [CPLEX](https://www.ibm.com/products/ilog-cplex-optimization-studio) or [Gurobi](https://www.gurobi.com/). Both of these solvers have free academic license programs.
+
+#### Connect CPLEX to CNApy
+
+*Note: This connection is only possible if you've installed CNApy according to the first main method, i.e. by using conda*
+
+Once installed CPLEX on your system (as described on [CPLEX's website](https://www.ibm.com/products/ilog-cplex-optimization-studio)), use conda in order to activate CNApy's conda environment:
```

Comment:
I would rephrase it like this: 
Once installed CPLEX on your system (as described on [CPLEX's website](https://www.ibm.com/products/ilog-cplex-optimization-studio)), the CPLEX Python API must be installed in the CNApy conda environment.

Also instead of: your CNApy's conda environment 
I would write: the CNApy conda environment